code/modules/cargo/packs/armory.dm

```diff
@@ -164,17 +164,6 @@
 					/obj/item/shield/riot)
 	crate_name = "riot shields crate"
 
-/datum/supply_pack/security/armory/riotshotguns
```

Comment:
They can get riot shields, riot gear, sec gear, detective stuff, ect ect. They are meant to replace gear, that gets lost, or broken, They are meant to order more gear when needed. This frankly being removed is rather insulting